telemetry/aggregator.go

```diff
@@ -2,23 +2,29 @@ package telemetry
 
 import (
 	"database/sql"
-	"fmt"
-	"log"
 	"time"
+
+	"go.uber.org/zap"
 )
 
 type Aggregator struct {
-	DB *sql.DB
+	DB     *sql.DB
+	logger zap.Logger
 }
 
-func NewAggregator(db *sql.DB) *Aggregator {
-	return &Aggregator{
-		DB: db,
+func NewAggregator(db *sql.DB) (*Aggregator, error) {
+	logger, err := zap.NewDevelopment()
```

Comment:
Probably we should have a single logger instance and pass it down like if it were a context.